sdk/edgeorder/Azure.ResourceManager.EdgeOrder/api/Azure.ResourceManager.EdgeOrder.net8.0.cs

```diff
@@ -232,44 +279,214 @@ protected MockableEdgeOrderSubscriptionResource() { }
 }
 namespace Azure.ResourceManager.EdgeOrder.Models
 {
+    public partial class AdditionalConfiguration : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.EdgeOrder.Models.AdditionalConfiguration>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.EdgeOrder.Models.AdditionalConfiguration>
```

Comment:
let us prepend RP name to this.

Reply:
Renamed to `ProductDetailsAdditionalConfiguration`